src/constants/screens.js

```diff
@@ -1,6 +1,8 @@
+/** @typedef {'2xl' | 'xl' | 'lg' | 'md' | 'sm' | 'mob'} Breakpoints */
+
 /**
  * Mapping of a breakpoint name to the lower-bound of its screen-width range
- * @type {Map<string, number>}
+ * @type {Map<Exclude<Breakpoints, 'mob'>, number>}
```

Comment:
!TIL we can declare types with Exclude!

Reply:
In the JS files all the great up-to-date TS features will be available: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Inside the `{}` basically everything that would fit on the right hand side of a TypeScript `type X =` statement is valid. JSDoc has some extra goodies (if you can call them that) to make up for limitations in the syntax, like `const` which allows applying the `as const` notation to an object.